apps/dav/lib/Capabilities.php

```diff
@@ -17,12 +17,13 @@ public function __construct(
 	}
 
 	/**
-	 * @return array{dav: array{chunking: string, bulkupload?: string, absence-supported?: bool, absence-replacement?: bool}}
+	 * @return array{dav: array{chunking: string, public-shares-chunking: bool, bulkupload?: string, absence-supported?: bool, absence-replacement?: bool}}
 	 */
 	public function getCapabilities() {
 		$capabilities = [
 			'dav' => [
 				'chunking' => '1.0',
+				'public-shares-chunking' => true,
```

Comment:
This is really we should try to avoid dashes in names that later will be available as json.
```suggestion
				'public_shares_chunking' => true,
```